docs/internal-documentation/distrib-testing/Ubuntu-Node.js-build-environment.md

```diff
@@ -27,14 +27,14 @@ here.
    source ~/.bashrc
    ```
 
-5. Install the supported Node.js version
+5. Install the Node.js version that supports Zonemaster GUI
    ```sh
-   nvm install 16
+   nvm install 16.18.0
```

Comment:
We have not strong requirement on the minor version and so I think we should not mention it (and definitely not mention the patch version, in case of security upgrade). The latest minor should work well.